admin/lib/solidus_admin/container.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "dry/system"
+require "dry/system/container"
+require "dry/system/component"
```

Comment:
I'm not against adding this dependency, but just to be aware of all the pieces of the tradeoff we are doing here, do you have an idea of the effort needed to create something similar internally? I have the feeling it's not a giant effort, and maybe it might make sense only if we use small parts of what's provided in dry container/component; interested in your take on this.

Reply:
Makes sense, let's re-evaluate if we really need it once we are ok with the architecture and we see it in action. Thanks!